packages/eslint-config-custom/package.json

```diff
@@ -7,10 +7,11 @@
     "@typescript-eslint/eslint-plugin": "5.47.1",
     "@typescript-eslint/parser": "5.47.1",
     "eslint": "8.30.0",
-    "eslint-config-next": "13.1.1",
     "eslint-config-airbnb": "19.0.4",
     "eslint-config-airbnb-typescript": "17.0.0",
+    "eslint-config-next": "13.1.1",
     "eslint-config-prettier": "8.5.0",
+    "eslint-import-resolver-typescript": "^3.5.3",
```

Comment:
Pin dependencies 😛

Yarn supports an argument `-E` to install dependencies with exact version.
e.g:
```
yarn add -DE eslint-import-resolver-typescript
```

Reply:
Oh I didn't knew, great!